docs/source/azure/examples/teamsnewresourcegroup-general-notifications.rst

```diff
@@ -0,0 +1,58 @@
+.. _azure_examples_teams_new_resource_group:
+
+Resource Group - Generate a Teams Message on Create
+===================================================
+
+This policy will send a notification to a Microsoft Teams channel when
+a new resource group is created in the subscription. There is a few minute 
+delay between the resource group being created and the notification appearing.
+
+In order to target the correct Teams channel, you will need to insert the 
+custom incoming webhook for the channel. If you do not have a webhook set up,
+please see `Setting up a custom incoming webhook <https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/connectors/connectors-using#setting-up-a-custom-incoming-webhook>`_.
+
+We are using an `actionable message card <https://docs.microsoft.com/en-us/outlook/actionable-messages/send-via-connectors>`_ 
+to provide a link to the new resource group. Note: linking to portal pages within 
+Azure is not supported by all web browsers.
+
+To run the policy, you must replace ``<your_webhook_here>`` with the correct url.
+
+For more information on how the event grid function works please see :ref:`azure_hostingoptions`
+
+.. code-block:: yaml
+
+    policies:
+      - name: notify-new-resource-group
+        description: |
+          Generates a Teams notification when a new resource group is created
+        resource: azure.resourcegroup
+        mode:
+          type: azure-event-grid
+          events: [{
+            resourceProvider: Microsoft.Resources/subscriptions/resourceGroups,
+            event: write
+          }]
+        actions:
+          - type: webhook
+            url: <your_webhook_here>
+            batch: false
+            body: >
+              resource.{
```

Comment:
Might be worth changing this slightly based on a previous customers issues.  People always want to reference some of the other fields that are siblings to resource, but looking at parents is impossible  in JMESPath.  

Instead of jumping into `resource.` right here you can just open with the `{` and reference `resource.location` and `resource.id`, etc, in your text.   Then you would have access to `account_id` to mention the subscription in question.